client/scala/src/main/scala/ai/verta/client/package.scala

```diff
@@ -0,0 +1,6 @@
+package ai.verta
+
+import scala.concurrent.ExecutionContext
+
+package object client {
```

Comment:
Don't use package objects.

Reply:
oh i already removed the methods inside of this but for some reason forgot to remove the actual file. resolved.